modules/sourcekitten/0.37.0/presubmit.yml

```
@@ -0,0 +1,25 @@
+shell_commands: &shell_commands
+- "echo --- Downloading and extracting Swift $SWIFT_VERSION to $SWIFT_HOME"
+- "mkdir $SWIFT_HOME"
+- "curl https://download.swift.org/swift-${SWIFT_VERSION}-release/ubuntu2004/swift-${SWIFT_VERSION}-RELEASE/swift-${SWIFT_VERSION}-RELEASE-ubuntu20.04.tar.gz | tar xvz --strip-components=1 -C $SWIFT_HOME"
+
+tasks:
+  verify_targets_linux:
+    name: Verify targets (Linux)
+    platform: ubuntu2004
+    environment:
+      CC: "clang"
+      SWIFT_VERSION: "5.7.2"
+      SWIFT_HOME: "$HOME/swift-$SWIFT_VERSION"
+      PATH: "$PATH:$SWIFT_HOME/usr/bin"
+    shell_commands: *shell_commands
+    build_flags:
+      - "--action_env=PATH"
+    build_targets:
+      - '@sourcekitten//:sourcekitten'
+  verify_targets_macos:
+    name: Verify targets (macOS)
+    platform: macos
+    build_targets:
+      # TODO: Build `:sourcekitten` target when CI has Xcode 14 installed
+      - '@sourcekitten//:SourceKittenFramework'
```

According to the [previous release](https://github.com/bazelbuild/bazel-central-registry/pull/2504/commits/2e20a9d4f0d79cd01e25b42a71179622e2545d60):
```suggestion
tasks:
   verify_targets:
     name: Verify build targets
     platform: macos
     bazel: 7.x
     build_targets:
       - '@sourcekitten//:sourcekitten'
     build_flags:
       - "--repo_env=BAZEL_DO_NOT_DETECT_CPP_TOOLCHAIN=1"
```